docs/pages/guides/adding-delegation.mdx

```diff
@@ -429,7 +430,7 @@ const publicClient = createPublicClient({
 //   client - a wallet client that uses a random account
```

Comment:
we're not returning this anymore, so I don't think we need this comment?

Reply:
and that said, should we just return the `getContract` result rather than `{ world: getContract(...) }`?